src/wp-admin/authorize-application.php

```diff
@@ -194,24 +201,24 @@
 		}
 		?>
 
-		<?php if ( $new_password ) : ?>
-			<div class="notice notice-success notice-alt below-h2">
-				<p class="application-password-display">
-					<label for="new-application-password-value">
-						<?php
-						printf(
-							/* translators: %s: Application name. */
-							esc_html__( 'Your new password for %s is:' ),
-							'<strong>' . esc_html( $app_name ) . '</strong>'
-						);
-						?>
-					</label>
-					<input id="new-application-password-value" type="text" class="code" readonly="readonly" value="<?php esc_attr( WP_Application_Passwords::chunk_password( $new_password ) ); ?>" />
-				</p>
-				<p><?php _e( 'Be sure to save this in a safe location. You will not be able to retrieve it.' ); ?></p>
-			</div>
+		<?php
+		if ( $new_password ) :
+			$message = '<p class="application-password-display">
+				<label for="new-application-password-value">' . sprintf(
+				/* translators: %s: Application name. */
+				esc_html__( 'Your new password for %s is:' ),
+				'<strong>' . esc_html( $app_name ) . '</strong>'
+			) . '
+				</label>
+				<input id="new-application-password-value" type="text" class="code" readonly="readonly" value="' . esc_attr( WP_Application_Passwords::chunk_password( $new_password ) ) . '" />
+			</p>
+			<p>' . __( 'Be sure to save this in a safe location. You will not be able to retrieve it.' ) . '</p>';
+			$args = array(
+				'type'               => 'success',
+				'additional_classes' => array( 'notice-alt', 'below-h2' ),
```

Comment:
I think we might need `'paragraph_wrap' => false` here as there's already `<p>` tags included in the message.